### DISCUSSION TITLE
新增删除节点局部更新

### DISCUSSION THREAD

**Discussion by xlys997** [antvis/G6#6640] [Q&A]
g6 新增/删除一个节点，如何实现局部更新
draw  只绘制，不更新布局
render 全部更新了，不想全部更新，只想局部更新

**Comment by Aarebecca** [comment]
后续会完善相关内容

**Reply by hustcc** [reply]
@Aarebecca 现在有解法吗？

**Reply by Aarebecca** [reply]
得排期增加相关机制

**Reply by code-lixm** [reply]
同求，很急，现在新增一个节点，就需要重新render一遍，同时addChildren和addData的行为不一致，addData不会更新父节点，addChildren可以，但是addChildren不能新增edge

**Reply by Leona-Li** [reply]
麻烦问下有具体支持的时间吗？不能局部更新，大数据量下添加/删除节点耗时很久
